docs/self-managed/console-deployment/configuration.md

```diff
@@ -0,0 +1,77 @@
+---
+id: configuration
+title: "Configuration"
+sidebar_label: "Configuration"
+description: "Read details on the configuration variables of Console Self-Managed."
+---
+
+:::note
+Console Self-Managed is available only to [enterprise customers](../../reference/licenses.md#console).
+:::
+
+Console Self-Managed can be configured using environment variables and configuration parameters.
+
+## Environment Variables
+
+| Environment variable                | Description                                                 | Example value                            |
+| ----------------------------------- | ----------------------------------------------------------- | ---------------------------------------- |
+| `KEYCLOAK_BASE_URL`                 | Base URL for Keycloak                                       | https://example.com/auth                 |
+| `KEYCLOAK_INTERNAL_BASE_URL`        | Internal Base URL for Keycloak                              | http://camunda-platform-keycloak:80/auth |
+| `KEYCLOAK_REALM`                    | Realm for Keycloak                                          | camunda-platform                         |
+| `CAMUNDA_IDENTITY_AUDIENCE`         | Audience for Console client                                 | console                                  |
+| `CAMUNDA_IDENTITY_CLIENT_ID`        | Client Id for Console client                                | console                                  |
+| `CAMUNDA_CONSOLE_CONTEXT_PATH`      | Context path for Console                                    | console                                  |
+| `CAMUNDA_CONSOLE_CUSTOMERID`        | Unique identifier of the customer                           | `customer-id`                            |
+| `CAMUNDA_CONSOLE_INSTALLATIONID`    | Unique installation id of the current customer installation | `installation-id`                        |
+| `CAMUNDA_CONSOLE_TELEMETRY_ENABLED` | Boolean flag to enable telemetry for Console Self-Managed   | `true`                                   |
+
+Console environment variables could be set in Helm via the `console.env` key. For more details, check [Console Helm values](https://artifacthub.io/packages/helm/camunda/camunda-platform#console-parameters).
+
+## Telemetry
+
+You can enable Telemetry and usage collection to help us improve our product by sending several telemetry metrics to Camunda. The information we collect will contribute to continuous product enhancement and help us understand how Camunda is used. We do not collect sensitive information and limit data points to several metrics. For more information, you can download collected data set metrics from the Telemetry page at anytime.
```

Comment:
"from the Telemetry page at anytime" -- should link to this page here.

Reply:
@christinaausley In this case the telemetry page in console self-managed is referenced so it's not possible to link to this page. But because you thought it's the case: maybe there's a way to improve the sentence and make it more clear?

Reply:
Wait, so we are referencing the UI? If so, we just need to bold it, but maybe I am misunderstanding.